miku-server/src/main/scala/org/http4s/miku/MikuBuilder.scala

```diff
@@ -0,0 +1,492 @@
+package org.http4s.miku
+
+import java.io.FileInputStream
+import java.net.InetSocketAddress
+import java.security.{KeyStore, Security}
+import javax.net.ssl.{KeyManagerFactory, SSLContext, TrustManagerFactory}
+
+import cats.effect.{Effect, IO}
+import com.typesafe.netty.HandlerPublisher
+import com.typesafe.netty.http.HttpStreamsServerHandler
+import fs2.Pipe
+import fs2.interop.reactivestreams._
+import io.netty.bootstrap.Bootstrap
+import io.netty.channel._
+import io.netty.channel.epoll.{Epoll, EpollEventLoopGroup, EpollServerSocketChannel}
+import io.netty.channel.group.DefaultChannelGroup
+import io.netty.channel.nio.NioEventLoopGroup
+import io.netty.channel.socket.nio.NioServerSocketChannel
+import io.netty.handler.codec.http.{
+  HttpContentDecompressor,
+  HttpRequestDecoder,
+  HttpResponseEncoder
+}
+import io.netty.handler.ssl.SslHandler
+import io.netty.handler.timeout.IdleStateHandler
+import org.http4s.HttpService
+import org.http4s.server._
+import org.http4s.util.bug
+import org.log4s.getLogger
+
+import scala.collection.JavaConverters._
+import scala.collection.immutable
+import scala.concurrent.ExecutionContext
+import scala.concurrent.duration.Duration
+
+sealed trait NettyTransport
+case object Jdk extends NettyTransport
```

Comment:
Doesn't Netty call this `Nio`?

For namespacing, maybe put these in a companion object of `NettyTransport`, since the names don't mean much on their own?

Reply:
Will do